control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -347,6 +347,15 @@ func (r *Reconciler) finalizeKind(ctx context.Context, broker *eventing.Broker)
 		}
 	}
 
+	if secret != nil {
```

Comment:
we did log it before we did differentiate between external/internal, hence doing this now again